gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/avatarpicker/AvatarPicker.kt

```diff
@@ -163,7 +164,8 @@ internal fun AvatarPicker(
     }
 }
 
-private const val UCROP_COMPRESSION_QUALITY = 100
+private const val UCROP_COMPRESSION_QUALITY = 90
+private const val UCROP_MAX_IMAGE_SIZE = 1080
```

Comment:
Set this as the max size for now - seems enough for an Avatar though happy to hear your opinions.